prime-dotnet-webapi/Migrations/20200319180139_ValidateLicenceFlag.cs

```diff
@@ -0,0 +1,331 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class ValidateLicenceFlag : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
+                table: "DefaultPrivilege",
+                keyColumns: new[] { "PrivilegeId", "LicenseCode" },
+                keyValues: new object[] { 18, 16 });
```

Comment:
Could you make this an UpdateData rather than a drop and re-add?

Reply:
It is removing one privilege and replacing it with another so i dont think it needs to be an update in this case.